src/desktop/modules/manager/contentImportManager/contentImportHelper.ts

```diff
@@ -4,7 +4,7 @@ import * as  fs from "fs";
 import * as  _ from "lodash";
 import * as path from "path";
 import { manifest } from "../../manifest";
-import { containerAPI, ISystemQueue } from "@project-sunbird/OpenRAP/api";
+import { containerAPI, ISystemQueue } from "../../../../desktop/OpenRAP/dist/api";
```

Comment:
this should be @project-sunbird/OpenRAP/api

Reply:
fixed